site/content/en/docs/technical-outcomes/scalability/scale-using-vpa/scale-using-vpa.demo

```diff
@@ -0,0 +1,38 @@
+# Scale pod CPU request up & down using VPA
+
+# Get metrics usage
+wget https://github.com/kubernetes-sigs/kwok/releases/download/v0.6.0/metrics-usage.yaml
+
+# Set up Cluster
+kwokctl create cluster --enable-metrics-server --config ${WORK_DIR}/kwokctl.yaml --config ./metrics-usage.yaml --runtime binary
+
+# Create a node
+kwokctl scale node --replicas 1 --param '.allocatable.cpu="4000m"'
+
+# Set up VPA
+git clone https://github.com/kubernetes/autoscaler.git
```

Comment:
I think this can be removed in the demo, and in the docs that need to clone this repo

Reply:
The commands in Lines 14-20 may fail if it's removed.

Reply:
As a note in the docs, is need to clone the repo ahead

Reply:
Ok. How about when I'm regenerating the SVG?